crates/router/src/connector/paypal/transformers.rs

```diff
@@ -1412,10 +1671,13 @@ impl<F>
             status,
             response: Ok(types::PaymentsResponseData::TransactionResponse {
                 resource_id: types::ResponseId::ConnectorTransactionId(item.response.id),
-                redirection_data: Some(paypal_threeds_link((
+                redirection_data: match paypal_threeds_link((
                     link,
                     item.data.request.complete_authorize_url.clone(),
-                ))?),
+                )) {
+                    Ok(url) => Some(url),
+                    Err(_) => None,
+                },
```

Comment:
Are we okay with doing nothing when we receive an error? Can we at least log it?